src/controllers/user.controller.js

```diff
@@ -163,7 +192,7 @@ const editUserProfile = async(req,res)=>{
     const userEmail = req.user.email
     const decodeUser =await findUserByEmail(userEmail);
 
-    if(!decodeUser) return res.status(401).json("user not found");
+    if(!decodeUser) return res.status(401).json('user not found');
```

Comment:
1. format this response as well
2. 401 status code if for Unauthorized NOT not found